receiver/dockerstatsreceiver/factory.go

```diff
@@ -20,29 +20,17 @@ import (
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer"
-	"go.opentelemetry.io/collector/featuregate"
 	rcvr "go.opentelemetry.io/collector/receiver"
 	"go.opentelemetry.io/collector/receiver/scraperhelper"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dockerstatsreceiver/internal/metadata"
 )
 
 const (
-	typeStr        = "docker_stats"
-	stability      = component.StabilityLevelAlpha
-	useScraperV2ID = "receiver.dockerstats.useScraperV2"
+	typeStr   = "docker_stats"
+	stability = component.StabilityLevelAlpha
 )
 
-func init() {
```

Comment:
so, to make my previous comment clear, I am fine with merging this PR if we don't remove this; everything else can be merged now IMO